package.json

```diff
@@ -5,23 +5,23 @@
   "module": "src/index.js",
   "main": "dist/index.js",
   "browser": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "src",
     "dist"
   ],
   "scripts": {
     "clean:dist": "rimraf ./dist/*",
     "clean:docs": "rimraf build/docs",
-    "lint:docs": "documentation lint src/index.js",
     "lint:src": "eslint --ext js src/",
-    "lint": "npm run lint:docs && npm run lint:src",
+    "lint": "tslint --project tsconfig.json --config tslint.json && tsc --noEmit --project tsconfig.json",
     "pretest": "npm run lint",
-    "test": "jest --silent --maxWorkers=4",
+    "test": "jest",
+    "test:ci": "jest --ci --coverage",
     "test:watch": "jest --watchAll",
-    "start:docs": "documentation serve -w -g src/index.js",
-    "build:docs": "npm run clean:docs && documentation build -g -f html -o build/docs src/index.js",
-    "build:docs-md": "documentation readme src/index.js --section=\"API Documentation\"",
-    "build:dist": "npm run clean:dist && BABEL_ENV=production babel src --out-dir dist --copy-files --ignore spec.js",
+    "build": "npm run test:ci && npm run build:dist && npm run build:docs",
+    "build:docs": "npm run clean:docs && typedoc --out ./doc/ --exclude **/*.spec.**",
+    "build:dist": "tsc -p ./tsconfig.prod.json",
     "coveralls": "cat coverage/lcov.info | coveralls",
     "prepublishOnly": "npm run build:dist",
     "deploy": "NODE_DEBUG=gh-pages node tasks/update-gh-pages.js",
```

Comment:
I assume it should not be a problem, but have you tried if the task is still working with the new documentation framework?

Reply:
I change the outputdir to `./build/docs` so it should fit the ghpages task now.